src/calendareventmodification.h

```diff
@@ -41,34 +41,27 @@
 #include "calendarchangeinformation.h"
 #include "calendarcontactmodel.h"
 
-class CalendarEventModification : public QObject
+class CalendarEventModification : public CalendarEvent
```

Comment:
Was assuming for a moment that these are redundant Q_PROPERTYs but then realized there's also WRITE methods declared.  Perhaps that could also warrant some note.

Reply:
I've added a note mentioning the overloading with the write definition.